### PR TITLE
UX changes

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#6086

*Description of changes:*
- UX changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/new-request/search.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
                   <v-tooltip :disabled="!data.item.blurbs" right transition="fade-transition">
                     <template v-slot:activator="scope">
                       <span v-on="scope.on" class="list-item">
-                        {{ displayRequestActionListItem(data.item) }}
+                        {{ data.item.text }}
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:33]: Why did you revert this change?

File: client/src/components/new-request/search.vue [hunk 2]
```diff
@@ -419,11 +422,8 @@ export default class NewSearch extends Vue {
   clearErrors () {
     newReqModule.clearErrors()
   }
-  displayRequestActionListItem (item) {
-    if (item && item.text) {
-      return item.text.replace('Relocate into', 'Relocate into BC')
-    }
-    return ''
+  setClass (value) {
+    return value ? 'disabled-custom' : ''
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:426]: Did you know you can have this logic in the `:class` property?

See
https://github.com/bcgov/namerequest/blob/1a25b0db8ee1389e89c7dd0392835d35706dd50a/client/src/components/tabs.vue#L15
or
https://github.com/bcgov/namerequest/blob/1a25b0db8ee1389e89c7dd0392835d35706dd50a/client/src/components/common/contact-info.vue#L2

File: client/src/components/new-request/search.vue (hunk 1 above)
Comment (kialj876) [reply client/src/components/new-request/search.vue:33]: it was a typo in the UX design to begin with :P

Comment (kialj876) [reply client/src/components/new-request/search.vue:33]: The design had it dynamically changing from dropdown list to selection, but it was just a mistake

File: client/src/components/new-request/search.vue (hunk 2 above)
Comment (kialj876) [reply client/src/components/new-request/search.vue:426]: oooh cool okay I will update

**Review by severinbeauvais — APPROVED** [review]
Looks great! (I tested locally.)

File: client/src/components/new-request/search.vue (hunk 2 above)
Comment (severinbeauvais) [reply client/src/components/new-request/search.vue]: +1 for sexy code

**Review by cameron-freshworks — APPROVED** [review]
Looks good!
